pipeline/commands/landing.py

```diff
@@ -64,7 +67,8 @@ def landing(token: Optional[str] = None):
         "base_key": base_key,
         "bucket_name": working_bucket,
         "reconstruction_id": reconstruction_id,
-        "run_id": run_id
+        "run_id": run_id,
+        "run_tilt": (metadata['slice_transform'] is not None)
```

Comment:
I wish we did not add this into the landing task, because now the tasks serves a dual purpose.
But I personally do not know if there is a better way. You would need to update the docstring for the method to reflect change. Perhaps wrapping logic for run_tilt into a function would make it better as well.

Reply:
The other alternative is to have the tilt command not run if it's passed a None for the slice transform. I think the landing task always will serve the purposes of unpacking and preparing for the run, so I don't see the conflict.

Reply:
I like having this sort of thing in the landing task. We could maybe rename to "has_slice_transform" or "requires_tilt_correction"